public/tutorials/setup-for-erc20-upgrade.sh

```
@@ -0,0 +1,76 @@
+#! /bin/sh
+
+rm -rf upgrade-erc20
+mkdir upgrade-erc20
+cd upgrade-erc20
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)